client/components/controllers/GameEnded.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const GameEnded = props => {
+  return (
+    <div className="container">
+      <div className="row center">
+        <h5 className="card-panel blue lighten-4 s12">Thank you for playing</h5>
```

Comment:
Again, if you remove the ROW above, the s12 here becomes unnecessary, the div will take up the whole width.